api/services/ocean/spark/schemas/oceanSparkJob.yaml

```diff
@@ -0,0 +1,55 @@
+type: object
+required:
+- id
+- displayName
+- accountId
+- organizationId
+- clusterId
+- controllerClusterId
+description: >
+  The description of an Ocean Spark job
+properties:
+  internalId:
+    $ref: "internalId.yaml"
+  id:
+    $ref: "oceanSparkJobId.yaml"
+  displayName:
+    $ref: "oceanSparkJobDisplayName.yaml"
+  controllerClusterId:
+    $ref: "controllerClusterId.yaml"
+  clusterId:
+    $ref: "oceanSparkClusterId.yaml"
+  accountId:
+    $ref: "accountId.yaml"
+  organizationId:
+    $ref: "organizationId.yaml"
```

Comment:
@HHK1 @arnarpall those fields are clear from the context, so we could remove them from the API.
I think you noted this down already, but I just wanted to confirm.
I wonder if it's clear from the context in all usages of the data class though.

(same question as for the applications)

Reply:
I will remove those two fields from the doc, will be implemented in spotinst.atlassian.net/browse/BGD-1669